ic-utils/src/interfaces/management_canister/builders.rs

```diff
@@ -447,10 +462,8 @@ impl<'agent, 'canister: 'agent> InstallCodeBuilder<'agent, 'canister> {
     }
     /// Set the argument with multiple arguments as tuple to the installation,
     /// which will be passed to the init method of the canister. Can be called at most once.
-    pub fn with_args(mut self, tuple: impl candid::utils::ArgumentEncoder) -> Self {
-        if self.arg.0.is_some() {
-            panic!("argument is being set more than once");
-        }
+    pub fn with_args(mut self, tuple: impl ArgumentEncoder) -> Self {
+        assert!(self.arg.0.is_none(), "argument is being set more than once");
```

Comment:
If I understand this, this means:
- in a debug build, calling with_args more than once will result in a panic
- in a release build, calling with_args more than once will silent replace the args

Is that how it is, and desired?

Reply:
No. That is `debug_assert!`. The replacement is exactly equivalent, just more idiomatic.